lib/simulator-xcode-6.js

```diff
@@ -569,13 +569,17 @@ class SimulatorXcode6 extends EventEmitter {
     // locally, 7.1 averages 8.5 retries (from 6 - 12)
     //          8 averages 0.6 retries (from 0 - 2)
     //          9 averages 14 retries
-    await retryInterval(20, 250, async () => {
-      if (await this.isFresh()) {
-        let msg = 'Simulator files not fully created. Waiting a bit';
-        log.debug(msg);
-        throw new Error(msg);
-      }
-    });
+    try {
+      await retryInterval(60, 250, async () => {
+        if (await this.isFresh()) {
+          let msg = 'Simulator files not fully created. Waiting a bit';
+          log.debug(msg);
+          throw new Error(msg);
+        }
+      });
+    } catch (err) {
+      log.warn(`Timeout waiting for simulator files to be created. Continuing`);
```

Comment:
it's possible that it could work even if we hit the timeout?

Reply:
Yes. That's why I added the catch and log. Previously there was an unhandled promise rejection.